keyboards/bbrfkr/dynamis/dynamis.c

```diff
@@ -0,0 +1,24 @@
+/* Copyright 2022 bbrfkr
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "dynamis.h"
+#include "pointing_device.h"
+extern const pointing_device_driver_t pointing_device_driver;
+
+report_mouse_t pointing_device_task_kb(report_mouse_t mouse_report) {
+    mouse_report.y = -mouse_report.y;
+    return mouse_report;
+}
```

Comment:
```suggestion
```
This shouldn't be needed, as defining `POINTING_DEVICE_INVERT_Y` in your config.h will accomplish this.

Reply:
Thanks! Deleted it